### DISCUSSION TITLE
About get student developer pack

### DISCUSSION THREAD

**Discussion by Azimdjan** [community/community#22748] [New to GitHub]
<p>I have already submitted. However still, i am not student developer.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>It sounds as though your application requires a manual review. In that case I’m afraid that it can take up to 3 weeks at this busy time for your application to be completed.</p>
<p>If it takes longer than that, please contact <a href="https://support.github.com/">support</a> directly.</p>

**Comment by Azimdjan** [comment]
<p>It is almost 3 weeks. Still I cannot get Student Developer Pack!!!</p>
